### ISSUE TITLE
[med] : Many pages: Chrome + Talkback: Incorrect state is defined for dropdown.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8557]
## Action Performed:
1. Using Chrome, open URL staging.new.expensify.com
2. Navigate to Setting using swipe gesture and press double tap on it.
3. Navigate to Preferences control using swipe gesture and press double tap on it.
4. Navigate to Drop down control using swipe gesture.
5. Observe the screen reader announcement.

## Expected Result:
Correct state should be defined for the dropdown. Screen Reader should announce "priority mode Most recent dropdown collapsed."

## Actual Result:
Incorrect state is defined for dropdown. When focus lands on the dropdown screen reader announced as "disabled"

**Other occurrences**
Same issue repro on PR #8737, #9118, #8299

## Workaround:
Yes

## Area issue was found in:
Preferences

## Failed WCAG checkpoints
4.1.2

## User impact:
Due to the incorrect state, screen reader user may not able to know about the usage of the button which create confusion for them.

## Suggested resolution:
Buttons should have a valid state defined so user can easily interact with them. Eg:  State: Collapsed/Expanded.

Refer to:
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

## Platform:
- Android

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/162372984-34aa8b96-14e7-4acf-ad95-2a88ef9818c2.mp4
